### ISSUE TITLE
--lycoris for z image?

### ISSUE THREAD

**Issue by DarkAlchy** [kohya-ss/musubi-tuner#884]
Are we going to eventually get this for z image?  AITK has lokr, and loha with a tweak, and Blue made his very modular eons ago.

Any chance?

**Comment by DarkAlchy** [comment]
Never mind, I added it.

import network module: networks.lora_zimage_lycoris
2026-01-31 00:22:51|[LyCORIS]-INFO: Using rank adaptation algo: loha
2026-01-31 00:22:51|[LyCORIS]-INFO: Use Dropout value: 0.0
2026-01-31 00:22:51|[LyCORIS]-INFO: Create LyCORIS Module
2026-01-31 00:22:52|[LyCORIS]-INFO: create LyCORIS: 276 modules.
2026-01-31 00:22:52|[LyCORIS]-WARNING: No modules matched the following target module classes: ['Conv1d', 'Conv2d', 'Conv3d', 'GroupNorm']
2026-01-31 00:22:52|[LyCORIS]-INFO: module type table: {'LohaModule': 276}
Z-Image: Gradient checkpointing enabled. CPU offload: False
prepare optimizer, data loader etc.
INFO:musubi_tuner.hv_train_network:use 8-bit AdamW optimizer | {'weight_decay': 0.02, 'betas': (0.9, 0.999)}
override steps. steps for 8 epochs is / 指定エポックまでのステップ数: 2880
running training / 学習開始
  num train items / 学習画像、動画数: 360
  num batches per epoch / 1epochのバッチ数: 360
  num epochs / epoch数: 8
  batch size per device / バッチサイズ: 1
  gradient accumulation steps / 勾配を合計するステップ数 = 1
  total optimization steps / 学習ステップ数: 2880

Thanks.

**Comment by bpxw** [comment]
Sorry, how did you add it?
